pkgs/development/python-modules/kinparse/default.nix

```
@@ -2,37 +2,46 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
-  pytest,
-  future,
   pyparsing,
+  pytestCheckHook,
+  setuptools,
 }:
 
-buildPythonPackage {
+buildPythonPackage rec {
   pname = "kinparse";
-  version = "unstable-2019-12-18";
-  format = "setuptools";
+  version = "1.2.3";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "xesscorp";
     repo = "kinparse";
-    rev = "eeb3f346d57a67a471bdf111f39bef8932644481";
-    sha256 = "1nrjnybwzy93c79yylcwmb4lvkx7hixavnjwffslz0zwn32l0kx3";
+    tag = version;
+    hash = "sha256-170e2uhqpk6u/hahivWYubr3Ptb8ijymJSxhxrAfuyI=";
   };
 
-  pythonImportsCheck = [ "kinparse" ];
+  # Remove python2 build support as it breaks python >= 3.13
+  postPatch = ''
+    substituteInPlace setup.cfg \
+      --replace-fail "universal = 1" "universal = 0"
+'';
```

```suggestion
  '';
```
(nixfmt not happy about this)